apps/desktop-wallet/locales/id-ID/translation.json

```diff
@@ -491,4 +491,6 @@
   "Collection": "Collection",
   "ID": "ID",
   "Copy ID": "Copy ID"
```

Comment:
Can you please add a comma (`,`) at the end of this line? Otherwise the file format is invalid. Thank you!